### PR TITLE
make bdr ddl replicate GA

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-bidirectional-replication.md [hunk 1]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
 
 为了能够解决上述可复制的 DDL 和不可复制的 DDL 两类 DDL 的同步问题，TiDB 引入了两种 BDR role：
 
-- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL，可复制的 DDL 会被 TiCDC 同步到下游。
-- `SECONDARY`：你不能执行可复制的 DDL，也不能执行不可复制的 DDL，但是可以执行从 TiCDC 同步过来的 DDL。
+- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 Primary 集群执行的可复制的 DDL 同步到下游。
+- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 Primary 集群接收到的 DDL 同步到 SECONDARY 集群。
```
Comment (hfxsd) [inline ticdc/ticdc-bidirectional-replication.md:93]:
```suggestion
- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群接收到的 DDL 同步到 SECONDARY 集群。
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 2]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
 
 为了能够解决上述可复制的 DDL 和不可复制的 DDL 两类 DDL 的同步问题，TiDB 引入了两种 BDR role：
 
-- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL，可复制的 DDL 会被 TiCDC 同步到下游。
-- `SECONDARY`：你不能执行可复制的 DDL，也不能执行不可复制的 DDL，但是可以执行从 TiCDC 同步过来的 DDL。
+- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 Primary 集群执行的可复制的 DDL 同步到下游。
```
Comment (hfxsd) [inline ticdc/ticdc-bidirectional-replication.md:92]:
```suggestion
- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 PRIMARY 集群执行的可复制的 DDL 同步到下游。
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 3]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群接收到的 DDL 同步到 SECONDARY 集群。
 
-在不设置 BDR role 时，你可以执行任意 DDL。但是在 TiCDC 开启 `bdr_mode=true` 之后，执行的 DDL 不会被 TiCDC 同步。
+在不设置 BDR role 时，你可以执行任意 DDL。但是处于 BDR 模式中的 changefeed 不会同步该集群上的任何 DDL。
 
-> **警告：**
->
-> 双向复制的 DDL 同步目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+总的来说，在 BDR 模式中，TiCDC 仅会同步 Primary 集群上的可复制的 DDL 到下游。
```
Comment (hfxsd) [inline ticdc/ticdc-bidirectional-replication.md:97]:
```suggestion
总的来说，在 BDR 模式中，TiCDC 仅会同步 PRIMARY 集群上的可复制的 DDL 到下游。
```

**Review by Frank945946 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-bidirectional-replication.md [hunk 4]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 PRIMARY 集群执行的可复制的 DDL 同步到下游。
+- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群接收到的 DDL 同步到 SECONDARY 集群。
 
-在不设置 BDR role 时，你可以执行任意 DDL。但是在 TiCDC 开启 `bdr_mode=true` 之后，执行的 DDL 不会被 TiCDC 同步。
+在不设置 BDR role 时，你可以执行任意 DDL。但是处于 BDR 模式中的 changefeed 不会同步该集群上的任何 DDL。
 
-> **警告：**
->
-> 双向复制的 DDL 同步目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+总的来说，在 BDR 模式中，TiCDC 仅会同步 PRIMARY 集群上的可复制的 DDL 到下游。
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:97]:
```suggestion
总的来说，在 BDR 模式中，TiCDC 仅会将 PRIMARY 集群上的可复制的 DDL 同步到下游。
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 5]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 PRIMARY 集群执行的可复制的 DDL 同步到下游。
+- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群接收到的 DDL 同步到 SECONDARY 集群。
 
-在不设置 BDR role 时，你可以执行任意 DDL。但是在 TiCDC 开启 `bdr_mode=true` 之后，执行的 DDL 不会被 TiCDC 同步。
+在不设置 BDR role 时，你可以执行任意 DDL。但是处于 BDR 模式中的 changefeed 不会同步该集群上的任何 DDL。
 
-> **警告：**
->
-> 双向复制的 DDL 同步目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+总的来说，在 BDR 模式中，TiCDC 仅会将 PRIMARY 集群上可复制的 DDL 同步到下游。
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:97]:
```suggestion
总的来说，在 BDR 模式下，TiCDC 仅会将 PRIMARY 集群上可复制的 DDL 同步到下游。
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 6]
```diff
@@ -89,14 +89,12 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
 
 为了能够解决上述可复制的 DDL 和不可复制的 DDL 两类 DDL 的同步问题，TiDB 引入了两种 BDR role：
 
-- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL，可复制的 DDL 会被 TiCDC 同步到下游。
-- `SECONDARY`：你不能执行可复制的 DDL，也不能执行不可复制的 DDL，但是可以执行从 TiCDC 同步过来的 DDL。
+- `PRIMARY`：你可以执行可复制的 DDL，但不能执行不可复制的 DDL。TiCDC 会把 PRIMARY 集群执行的可复制的 DDL 同步到下游。
+- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群接收到的 DDL 同步到 SECONDARY 集群。
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:93]: “接收到”是否要改为“执行”？看到英文 PR 里用的 executed
```suggestion
- `SECONDARY`：你不能执行上述所有的 DDL。但是，TiCDC 能够把 PRIMARY 集群执行的 DDL 同步到 SECONDARY 集群。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**